oval-schemas/oval-definitions-schema.xsd

```diff
@@ -938,6 +939,17 @@
             <xsd:group ref="oval-def:ComponentGroup"/>
         </xsd:sequence>
     </xsd:complexType>
+    <xsd:complexType name="MergeFunctionType">
+        <xsd:annotation>
+            <xsd:documentation>The merge function takes one or more components and merges them together into a single string value, optionally including a delimiter string. For example, if data from one registry multi-size value contains values of "abc" and "def", the merge function operated on those values would resolve to a string with the value of 'abcdef'. If an optional delimiter of ',' was used, the merge function would resolve to a value of 'abc,'def'.</xsd:documentation>
```

Comment:
REG_MULTI_SZ doesn't actually mean "multi-size", it means multiple null-terminated strings.

Reply:
I swear I know what I'm doing from time to time... probably would have been faster to have you write this spec from the start.